packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -28,30 +31,38 @@ class MiniCart extends Component {
         })
     };
 
-    static defaultProps = {
-        // TODO: remove when connected to graphql
-        data: mockData
-    };
+    constructor(...args) {
+        super(...args);
+        this.loadAsyncDependencies();
```

Comment:
2 notes:

1. Class constructors should never have side-effects.
2. Data fetching should be done in `componentDidMount`, which has been said frequently the closer we are to async react (Fiber) - https://reactjs.org/docs/react-component.html#componentdidmount.

Reply:
Correct on both counts. This was done in `componentDidMount` for a reason.